linux/keyman-config/keyman_config/install_window.py

```diff
@@ -62,6 +62,15 @@ def __init__(self, kmpfile, online=False, viewkmp=None, language=None):
         with tempfile.TemporaryDirectory() as tmpdirname:
             extract_kmp(kmpfile, tmpdirname)
             info, system, options, keyboards, files = get_metadata(tmpdirname)
+            if not keyboards:
+                # Likely not a keyboard .kmp file
+                logging.info("%s is not a Keyman keyboard package" % kmpfile)
+                dialog = Gtk.MessageDialog(viewkmp, 0, Gtk.MessageType.ERROR, Gtk.ButtonsType.OK,
+                    _("The file '{kmpfile}' is not a Keyman keyboard package!").format(kmpfile=kmpfile))
```

Comment:
Does this string get localized to crowdin?

Reply:
yes